src/Tribe/RSVP.php

```diff
@@ -1003,6 +1004,28 @@ public function delete_ticket( $event_id, $ticket_id ) {
 		return true;
 	}
 
+	/**
+	 * Trigger for tribe_get_cost if there are only RSVPs
+	 * @since TBD
+	 *
+	 * @param string $cost
+	 * @param int $post_id
+	 * @param boolean $unused_with_currency_symbol
+	 *
+	 * @return string $cost
+	 */
+	public function trigger_get_cost( $cost, $post_id, $unused_with_currency_symbol ) {
+
+		if (
+			empty( $cost )
+			&& tribe_events_has_tickets( get_post( $post_id ) )
+		) {
+			$cost = esc_html__( 'Free', 'event-tickets' );
```

Comment:
Just a question since I’m not at the computer, does this filter require you to escape the return? Wondering if this should be a plain __()

Reply:
It's a bit of a tossup, I've removed it because it should be text only and it's a hard-coded (though translated) string. It bails if there is anything but an empty string/value (when it should display "Free". The filter itself could likely use some escaping, it's not escaped before returning?

Reply:
Filter is here: https://github.com/moderntribe/the-events-calendar/blob/master/src/functions/template-tags/general.php#L830

Reply:
tribe_get_cost() usage elsewhere is probably / should be escaped, so yeah we can remove that escaping here and be good